classes/controllers/FrmGatedContentController.php

```
@@ -0,0 +1,242 @@
+
+	/**
+	 * Generate a gated content token when a form action fires.
+	 *
+	 * @param object $action Form action post object (post_excerpt = 'gated_content').
+	 * @param object $entry  Submitted form entry object.
+	 * @param object $form   Form object.
+	 * @param string $event  Trigger event ('create', 'payment-success', 'user_registration', …).
+	 *
+	 * @return void
+	 */
+	public static function trigger( $action, $entry, $form, $event ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$form'. (undefined)

(UnusedFormalParameter)